README.md

```diff
@@ -81,6 +85,20 @@ You can also:
   * Run the solution tests with `MIX_ENV=test mix test --only solution`
   * Run the external tests with `MIX_ENV=test mix test --only external`
 
+The application is an [umbrella app](https://elixir-lang.org/getting-started/mix-otp/dependencies-and-umbrella-projects.html). It means that it is split into several sub-projects (that you can see under `/apps`).
+
+To run tests for a specific app only(at least while we're on Elixir 1.8), use this:
```

Comment:
je rajouterais quelque chose du style :
To run tests for a specific app, for example the "transport" app (at least while we're on Elixir 1.8), use this command:

comme "transport" est un mot qu'on retrouve un peu partout, j'ai mis quelques secondes a comprendre que c'était --app <nomApp>

Reply:
Très bien vu ; je prépare une mise à jour avec vos remarques. Je vais aussi ajouter quelques trucs au glossaire.